picker-starter/src/plataform.ts

```diff
@@ -0,0 +1,18 @@
+import { OptionsParams, PickerPlatform } from './core'
```

Comment:
this filename is wrong `plataform` instead of `platform`

Reply:
Oh, thanks! 
I Mixed Portuguese and English 😆

Reply:
Fixed [here](https://github.com/storyblok/field-type-examples/pull/80/commits/df07585259f6a984b8c5e301b9cc26a8f7864008), @eunjae-lee.